wp-includes/components/class-wp-service-worker-navigation-routing-component.php

```diff
@@ -416,6 +441,8 @@ public function serve( WP_Service_Worker_Scripts $scripts ) {
 		}
 
 		$this->replacements = array(
+			'CACHING_STRATEGY'                 => wp_service_worker_json_encode( isset( $caching_strategy ) ? $caching_strategy : null ),
+			'CACHING_STRATEGY_ARGS'            => isset( $caching_strategy_args_js ) ? $caching_strategy_args_js : 'null',
```

Comment:
Shouldn't this be `null`, without quotes?

Reply:
No, because we're exporting JSON here. The `$caching_strategy_args_js` variable contains JS code, an IIFE which is used as a JS expression. If that variable is not set, then we need to output `null` to the service worker.